src/XMLimport.cpp

```diff
@@ -1785,12 +1785,11 @@ void XMLimport::readIntegerList(QList<int>& list, const QString& parentName)
                 if (Q_LIKELY(!numberText.isEmpty() && ok)) {
                     list << num;
```

Comment:
Do we also want to validate the value to be one that this version of Mudlet supports, perhaps with:
```suggestion
                    switch (num) {
                    case REGEX_SUBSTRING:
                        [[fallthrough]];
                    case REGEX_PERL:
                        [[fallthrough]];
                    case REGEX_BEGIN_OF_LINE_SUBSTRING:
                        [[fallthrough]];
                    case REGEX_EXACT_MATCH:
                        [[fallthrough]];
                    case REGEX_LUA_CODE:
                        [[fallthrough]];
                    case REGEX_LINE_SPACER:
                        [[fallthrough]];
                    case REGEX_COLOR_PATTERN:
                        [[fallthrough]];
                    case REGEX_PROMPT:
                        list << num;
                        break;
                    default:
                        mpHost->postMessage(qsl("[ ERROR ] - \"%1\" as a number when reading the 'regexCodePropertyList' element of the 'Trigger' or 'TriggerGroup' element \"%2\" cannot be understood by this version of Mudlet, is it from a later version? Converting it to a SUBSTRING type so the data can be shown but it will probably not work as expected.").arg(numberText, parentName));
                        list << REGEX_SUBSTRING; //Set it to the default type
                    }                        

```

Reply:
Good idea I guess :)